app/message-sender.js

```diff
@@ -0,0 +1,41 @@
+const MessageBase = require('./message-base')
+const messageSchema = require('./message-schema')
+const logTraceMessage = require('./log-trace-message')
```

Comment:
You probably need to change the name to something like 'app-insights-....' as in the future we may want to track excpetions, events, etc.

Reply:
Makes sense, I've moved all of the app insights stuff to it's own module so it's clear what it is.  Consuming code referencing that module can set the variable to whatever makes sense in their context but it will now be clearer it's app insights.